packages/gestalt/src/IconButton.jsdom.test.js

```diff
@@ -1,13 +1,58 @@
 // @flow strict
-import React from 'react';
+import React, { createRef } from 'react';
 import { render } from '@testing-library/react';
 import IconButton from './IconButton.js';
 
 describe('IconButton', () => {
-  it('forwards a ref to the innermost button element', () => {
-    const ref = React.createRef();
+  it('handles click', () => {
+    const mockOnClick = jest.fn();
+    const { getByRole } = render(
+      <IconButton accessibilityLabel="test" icon="add" onClick={mockOnClick} />
+    );
+    getByRole('button').click();
+    expect(mockOnClick).toHaveBeenCalled();
+  });
+
+  it('renders a button and forwards a ref to the innermost <button> element', () => {
+    const ref = createRef();
     render(<IconButton disabled accessibilityLabel="test" ref={ref} />);
     expect(ref.current instanceof HTMLButtonElement).toEqual(true);
+    // $FlowIgnore[prop-missing]
```

Comment:
@AlbertCarreras could we remove these flow ignores? For every one we add, we need to think about ways to avoid it. Looks like we do the same for `<Button / >`, so we should fix those as well:

```js
    expect(ref.current instanceof HTMLAnchorElement).toEqual(true);
    if (ref.current instanceof HTMLAnchorElement) {
      expect(ref.current?.href).toEqual('http://www.pinterest.com/');
    }
```

Reply:
DONE